README.md

```diff
@@ -2,24 +2,280 @@
+
+Proposals that are pushed back at any point can be modified and resubmitted by the author(s). However, the updated PR must pass through the complete approval process from the beginning to ensure full consideration of the revised request. A new PR for this purpose can be submitted if the author(s) so desire, but must be linked to the original PR in order to retain full information regarding prior concerns and suggestions.
+
+
+### General voting rules
+
+The philosophy and goal of the ASC is to reach consensus on all, if not most, issues. In the event that full consensus cannot be reached, the ASC will use the following rules:
+
+* Each active ASC Member will receive one vote. All votes are required to pass with 2/3 majority. A quorum (>50%) of Members must be present (either physically or virtually). Abstentions will not be counted in the total votes when calculating whether there is a 2/3 majority.
+
+* A Member may submit a vote _in absentia_ to the ASC Co-Chairs in advance of the meeting if attendance by their Representative or designated Alternate is not possible (e.g., a quarterly meeting that falls during a national holiday period), but each ASC Member can only cast one vote. This prevents a Member from voting on behalf of its own entity and acting as a proxy for another Member. _In absentia_ votes are counted by the Co-Chairs as if the Member were present (thus counting towards the meeting's quorum requirement), but are recorded as having been voted _in absentia_ in the official tally.
+
+"Straw Polls" are used in several places as a means of gauging community support for a proposal - e.g., in assessing acceptance for a new Provisional API. Straw polls are conducted on an informal basis through the placement of emojis on a "poll" comment of an Issue or PR. The poll comment consists of the following text:
+
+```
+Please use emoji reactions ON THIS COMMENT to indicate your position on this proposal.
+
+    * You do not need to vote on every proposal
+    * If you have no opinion, don't vote - that is also useful data
+    * If you've already commented on this issue, please still vote so
+          we know your current thoughts
+    * Not all proposals solve exactly the same problem, so we may end
+          up accepting proposals that appear to have some overlap
+
+This is not a binding majority-rule vote, but it will be a very
+significant input into the corresponding ASC decision.
+
+Here are the meanings for the emojis:
+
+    * Celebration / Hooray / Tada!: I support this so strongly that I
+          want to be an advocate for it
+    * Heart: I think this is an ideal solution
+    * Smiley face: I'd be happy with this solution
+    * Thumbs up: I'd tolerate this solution
```

Comment:
Recommend using `thumb up` 👍  as a synonym to `smiley face` 😃 
and use  `:neutral_face` 😐 for the 'tolerate this solution'

Reply:
done